packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -11,64 +11,46 @@ import { RootDataObject } from "./rootDataObject";
 
 /**
  * Events emitted from {@link IFluidContainer}.
- *
- * @remarks
- *
- * The following is the list of events emitted.
- *
- * ### "connected"
- *
- * The "connected" event is emitted when the `IFluidContainer` completes connecting to the Fluid service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void;
- * ```
- *
- * ### "dispose"
- *
- * The "dispose" event is emitted when the `IFluidContainer` is disposed, which permanently disables it.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void;
- * ```
- *
- * ### "disconnected"
- *
- * The "disconnected" event is emitted when the `IFluidContainer` becomes disconnected from the Fluid service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void;
- * ```
- *
- * ### "saved"
- *
- * The "saved" event is emitted when the `IFluidContainer` has local changes acknowledged by the service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void
- * ```
- *
- * ### "dirty"
- *
- * The "dirty" event is emitted when the `IFluidContainer` has local changes that have not yet
- * been acknowledged by the service.
- *
- * #### Listener signature
- *
- * ```typescript
- * () => void
- * ```
  */
 export interface IFluidContainerEvents extends IEvent {
-    (event: "connected" | "dispose" | "disconnected" | "saved" | "dirty", listener: () => void): void;
+    /* eslint-disable @typescript-eslint/unified-signatures */
```

Comment:
Maybe we should disable this rule globally?  I feel like our event signatures are the main place it hits, and I like what you've done here.

Reply:
If folks liked this change, I was going to suggest as much. We should be striving to document our events separately, and now we get reasonable generated docs for call signatures, so I think it's a good policy to move forward with.

Reply:
Created #12195